src/tileanimation.cpp

```diff
@@ -48,15 +33,9 @@ void TileAnimationParams::serialize(std::ostream &os, u16 protocol_version) cons
 	}
 }
 
-void TileAnimationParams::deSerialize(std::istream &is, u16 protocol_version)
+void TileAnimationParams::deSerialize(std::istream &is, u8 version)
```

Comment:
uh?

Reply:
Yes i wondered about this too, why is 'version' an argument?

Reply:
what I am wondering about here is why it was changed from `u16 protover` to `u8 ver`

Reply:
`TileAnimationParams` does not have its own serialized version, it is bound to the [TileDef](https://github.com/minetest/minetest/pull/6319/files#diff-70868aa6d6b96c0c1623c761500d23c4L218) version. Shall I include a separate version number here for consistency?

Reply:
Oh duh it's for future use ok, but previously it used 'protocol_version', why change?

Reply:
@paramat This protocol version was backwards-calculates from the TileDef serialisation version - see link above. This is not accurate, so either the real protocol version or the TileDef version should be used directly. I decided for the 2nd option, as it's only used in the tile definition.

Reply:
Ok.

Reply:
please keep u16, since introduction of TOSERVER_HELLO protocol version is sent as a u16 and this should be propagated everywhere